make/jdk/src/classes/build/tools/generatecharacter/GenerateCharacter.java

```diff
@@ -188,15 +197,21 @@ the values in the table can be preshifted (generally possible if the table
     // maskMirrored needs to be long, if up 16-bit
     private static final long maskMirrored          = 0x80000000L;
 
-    // bit masks identify the 8-bit property field described above, in B
+    // bit masks identify the 16-bit property field described above, in B
     // table
     private static final long
-        maskOtherLowercase  = 0x0100000000L,
-        maskOtherUppercase  = 0x0200000000L,
-        maskOtherAlphabetic = 0x0400000000L,
-        maskIdeographic     = 0x0800000000L,
-        maskIDStart         = 0x1000000000L,
-        maskIDContinue      = 0x2000000000L;
+        maskOtherLowercase  = 0x000100000000L,
+        maskOtherUppercase  = 0x000200000000L,
+        maskOtherAlphabetic = 0x000400000000L,
+        maskIdeographic     = 0x000800000000L,
+        maskIDStart         = 0x001000000000L,
+        maskIDContinue      = 0x002000000000L,
+        maskEmoji           = 0x004000000000L,
+        maskEmojiPresentation = 0x008000000000L,
+        maskEmojiModifier   = 0x010000000000L,
+        maskEmojiModifierBase = 0x020000000000L,
+        maskEmojiComponent  = 0x040000000000L,
+        maskExtendedPictographic = 0x080000000000L;
 
```

Comment:
Would be nice the have the '=' aligned in the same column as the ones above, or at least have the emoji ones align with each other.

Reply:
Thanks. Indentation aligned, and although in production `-string` is used so comments are not emitted, added information for `B` table in `GenerateCharacter.propertiesComments` for consistency.